pyproject.toml

```diff
@@ -5,6 +5,53 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "python-ipware"
+authors = [
+    {name = "Val Neekman", email = "info@neekware.com"},
+]
+description = "A Python package to retrieve user's IP address"
+requires-python = ">=3.7"
+dynamic = ["version", "readme"]
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+
+[project.urls]
+Documentation = "https://github.com/un33k/python-ipware#readme"
+Issues = "https://github.com/un33k/python-ipware/issues"
+Source = "https://github.com/un33k/python-ipware"
+Changelog = "https://github.com/un33k/python-ipware/blob/main/CHANGELOG.md"
+
+[project.optional-dependencies]
+dev = [
+  "ruff",
+  "coveralls~=3.3",
+  "coverage[toml]",
+  "twine"
+]
+
+[tool.setuptools]
+packages = ["ipware"]
+
+[tool.setuptools.dynamic]
+version = {attr = "ipware.__version__"}
+readme = {file = ["README.md"]}
```

Comment:
It's this line.  Needs to be `readme = {file = ["README.md"], content-type = "text/markdown"}`.  setuptools assumes RST still.

Reply:
No problem.  Sorry about that.  Always hard to debug a final stage like that.  hatch must handle the content type for me, I haven't needed to set it before.